EIPS/eip-867.md

```diff
@@ -0,0 +1,190 @@
+
+If accepted, an ERP could easily compile the block at while the changes are to take place, and the source of the actions (which would be the output of the script, in a standardized object output). These can be bundled with the client for seamless execution.
+
+
+## ERP Approval Process
+Approval of any ERP will require a mandatory minimum of 30-days between the time the ERP is submitted (or substantially modified) and adoption by clients, and ERPs affecting more funds or more addresses may prompt a considerably longer review period.  In practice, ERPs will often require more time because the will likely be bundled into other client upgrades that are on their own schedule.  However, the 30-day mandatory review period  guarantees a period for comment and auditing and prevents any last minute inclusions from being rushed into an upcoming upgrade.  
+
+Once an ERP is published (via a pull request made to the EIP repository), the approval process begins.  The author is encouraged to seek feedback from the following groups:
+
+- People or organizations who are directly affected by the issue (e.g. entities who will be refunded or were associated with the original incident)
+- People or organizations that have a demonstrable history within the Ethereum community that are unaffected by the issue but may serve as relatively impartial adjudicators
+
+ERPs that have strong support from both groups of people will be much more likely to succeed.  The first group, although not impartial, is critical to determine that among the directly affected parties there is broad agreement about the desired outcome.  The second group is critical to help verify (a) the conditions defined in the verification script accurately represent the human readable version, (b) that the verification script generates the State Change Object submitted with the ERP, and (c) that the people/orgs representing the “affected parties” is representative.  
+
+Positive, explicit, and broad support from both groups will be critical to the approval process; a lack of negative feedback is not sufficient.  It is the sole responsibility of the ERP author and affected parties to find support. Feedback that is not substantive should be dismissed, and feedback for the ERP process as a whole should be directed to this meta-EIP.  Community members, in general, are encouraged to reference these guidelines to ensure the process remains focused, objective, and expedient.  
+
+The ERP author may indicate in the ERP comments when he/she believes there is sufficient support and request a review by an EIP editor (see EIP-1 for details).  The editor(s) will mark the ERP as “Accepted” or indicate why the proposal cannot be accepted.  An ERP that is not accepted may be re-evaluated upon substantial improvement.  
```

Comment:
Are the EIP editors obliged to mark “Accepted” or indicate why the proposal cannot be accepted in a prompt manner?

Reply:
Can the EIP editors mark "Accepted" before hearing client team's agreements?

Reply:
I suppose not.  The purpose of having the author indicate when they felt the proposal was ready was to _reduce_ the load on EIP Editors.  Maybe I misunderstood, but I thought the process would go like this:

1. The ERP is submitted as a draft once the scope of the ERP and the general rationale behind it is clear.  This would include a verification script, justification, and SCO file.  If the author has already solicited feedback from peers, great that can be included.
2. The author has not already gone through some sort of peer review, they can attempt to contact people in the community, point them to the ERP, and ask them to confirm that their claims are factual, that their justification is accurately represented by the verification script, and that the verification script generates the same output as originally posted by the author and would accomplish the expected state changes.  The review should be factual in nature ("is" or "is not" rather than "should" or "should not")
3. Once the author believes they have "left no stone unturned", only then would they ask for the EIP Editors time to review the claim.  ERPs that cannot meet this minimum bar would not be considered.  

EIP Editors are, of course, not obliged to do anything (they are volunteers, I believe).  However, acceptance of this EIP would indicate that the community would prefer things not to linger once they have met the minimum requirements.

Reply:
@phiferd for one thing, now I think the current EIP editorship is not well documented.  It is my fault not reviewing EIP-1 hard when I joined this.  I could have at least asked, "what is this Ethereum philosophy"?

Now I'm trying to make sure that this draft is not creating an easier shortcut than the current EIP process. 

One thing I can point out is, "to fork or not to fork" is not factual.  Somehow in this draft, the question seems reduced to only factual aspects.

Reply:
Understood.  I will try to clarify that in the EIP.  Our intention was not to circumvent anything, more to say "hey everybody -- if and when we ever consider a recovery proposal, here's the process that will be used to make sure it's legit and not an attempt to claim someone else's funds".  That's the factual part.  

You are correct, the decision to fork is not a factual question.  However, this EIP was also supposed to serve to establish a norm that would be "normally" be followed (but not enforced).  So, if the ERP passes the factual test without any question, but we are still not planning on including it, my hope is that the community would look at that skeptically and say "wait a minute, what's going on here, why is this case being treated differently?"